`llvm[core]` now fails on macOS with 
```
...
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg2/packages/llvm_arm64-osx/share/llvm/usage
CMake Error at ports/llvm/portfile.cmake:370 (file):
  file RENAME failed to rename

    /Users/leanderSchulten/git_projekte/vcpkg2/packages/llvm_arm64-osx/bin/lib

  to

    /Users/leanderSchulten/git_projekte/vcpkg2/packages/llvm_arm64-osx/tools/llvm/lib

  because: No such file or directory

Call Stack (most recent call first):
  scripts/ports.cmake:168 (include)

```

--- a/ports/llvm/portfile.cmake
+++ b/ports/llvm/portfile.cmake
@@ -367,7 +367,10 @@ if(empty_dirs)
 endif()
 
 vcpkg_copy_tool_dependencies("${CURRENT_PACKAGES_DIR}/tools/${PORT}")
-file(RENAME "${CURRENT_PACKAGES_DIR}/bin/lib" "${CURRENT_PACKAGES_DIR}/tools/${PORT}/lib")
+if(EXISTS "${CURRENT_PACKAGES_DIR}/bin/lib")
+    file(MAKE_DIRECTORY "${CURRENT_PACKAGES_DIR}/tools/${PORT}")
+    file(RENAME "${CURRENT_PACKAGES_DIR}/bin/lib" "${CURRENT_PACKAGES_DIR}/tools/${PORT}/lib")
+endif()
 
 if(NOT DEFINED VCPKG_BUILD_TYPE OR VCPKG_BUILD_TYPE STREQUAL "debug")
     file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/include"

--- a/versions/l-/llvm.json
+++ b/versions/l-/llvm.json
@@ -1,7 +1,7 @@
 {
   "versions": [
     {
-      "git-tree": "003a0fb6d8e57ba9d52d1bd2fbd83973099a0dd8",
+      "git-tree": "3d195e85c468428553d0aed0eeb76e3714231e3a",
       "version": "17.0.2",
       "port-version": 1
     },

